src/Features/Core/Portable/Completion/CompletionItem.cs

```diff
@@ -188,5 +189,24 @@ public override string ToString()
         {
             return DisplayText;
         }
+
+        private List<string> tags = null;
+        public IEnumerable<string> Tags
```

Comment:
Glad Jason was able to clarify that this wasn't quite what we agreed on. :tongue: :heart: